test/conftest.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+import rclpy
+
+def pytest_sessionstart(session):
+    rclpy.init()
```

Comment:
Hier würde ich ein leerer Array übergeben, sonst nutzt der intern sys.argv

```suggestion
    rclpy.init([])
```

Reply:
Geht nicht:
rclpy.init([])
TypeError: init() takes 0 positional arguments but 1 was given

Reply:
So gehts:
```
rclpy.init(args=[])
```